r_code/group_B.R

```diff
@@ -0,0 +1,144 @@
+#############################
+## Load required libraries ##
+#############################
+
+library(DESeq2)
+library(ggplot2)
+library(data.table)
+
+############################
+## Sourcing the functions ##
+############################
+
+source("r_code/group_B_functions.R")
+
+#################################
+## Defining output directories ##
+#################################
+
+OUTPUT_DIR <- "results"
+create_directory(OUTPUT_DIR)
+
+DATA_DIR <- file.path(OUTPUT_DIR, "data")
+create_directory(DATA_DIR)
+
+RESULTS_DIR <- file.path(OUTPUT_DIR, "results")
+create_directory(RESULTS_DIR)
+
+#########################
```

Comment:
for this group, I think making a mock would be their task, so this code could be used as an example in setting up test maybe